package/yast2-core.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jan 27 20:17:56 UTC 2022 - Dirk Müller <dmueller@suse.com>
+
+- require glibc-locale for testsuite in cs_CS locale
```

Comment:
well, at first we will need some id for change like bugzilla number or feature.

another thing that this glibc locale quite slow down whole yast build as yast2-core is build at very early stage and it is quite big. And one more question. When it is build require, does it mean it will be needed also during runtime or not? as package is quite big.

Reply:
@jreidinger currently glibc-locale is always installed, so my change is a noop. I submitted a proposal to the factory mailinglist to switch to glibc-locale-base in the build environment so that the other 9000+ packages can build without the "quite big and slow" glibc-locale. 

yast2-core however is running its tests with cs_CZ.utf8 locale. unless we want to change that that explicit buildrequires is needed. 

there is no customer bsc or fate behind it so I submitted it as is. let me know if a bsc is needed anyway.